guides/common/modules/con_provisioning-templates.adoc

```diff
@@ -8,6 +8,15 @@ In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning templates* to view th
 You can create a template or clone a template and edit the clone.
 For help with templates, navigate to *Hosts* > *Provisioning templates* > *Create Template* > *Help*.
 
+ifdef::satellite[]
+Templates supported by {Team} are indicated by a {Team} icon.
+
+To show or hide unsupported templates, in the {ProjectWebUI} navigate to *Admininster > Settings*.
+On the *Provisioning* page, change the value of *Show unsupported provisioning templates* and click *Submit*.
```

Comment:
Provisioning *tab* perhaps?

Reply:
..., *set* the value of \*Show unsupported...\* to \`true\` and click \*Submit\*.